visualSHARK/views.py

```diff
@@ -144,7 +151,7 @@ def get_queryset(self):
 
 class TagViewSet(MongoReadOnlyModelViewSet):
     """API Endpoint for Tags."""
-
+    read_perm = 'view_files'
```

Comment:
Should be view_commits as it is tied to the vcs and commits, in the future we might separate this but for now view_commits is sufficient.